src/fastmath/ml/regression.clj

```diff
@@ -1246,3 +1271,22 @@
      {:dose ##Inf
       :p p
       :stderr ##Inf})))
+
+;; Model printing methods
+(defmulti ->string class)
+
+(defn lm->string [{:keys [names coefficients]}]
```

Comment:
This function contains only coefficients and their analysis. What about the rest? I mean residual analysis, R2, adj R2, etc...?

Reply:
Sorry about this one, I thought we had decided to start with just the coefficients and work from there.

Reply:
No worries. I thought it would be great to make it complete in this PR (if it's possible of course). I plan to release an alpha version after merging this PR.